### DISCUSSION TITLE
Background picture wallpaper

### DISCUSSION THREAD

**Discussion by zarg9** [termux/termux-app#3971] [Ideas]
Hey, is it possible to add a background picture wallpaper for termux?

ex, user put a picture in ~/.termux/background.jpg

and there should be just the need to add an XML tag in the android termux Textview, like:

android:background="@drawable/myResouce"

the resource pointing the jpg file.
Thanks.
(I don't have a PC with Android Studio right now, and termux-app don't build with AIDE)

**Comment by agnostic-apollo** [comment]
#3079

**Reply by snooppr** [reply]
> #3079

Updated termux style and termux app 118.1. fdroid. This setting: choosing images for the background function is not present. The background wallpaper function is for the Termux/Github version, Termux/Fdroid is not present?
